src/test/wdio.conf.js

```diff
@@ -7,7 +7,7 @@ const waitForTimeouts = parseInt(process.env.DEFAULT_TIME_OUT) || 3000;
 const seleniumStandaloneArgs = {
   drivers: {
     chrome: {
-      version: '98.0.4758.102',
+      version: process.env.CHROME_DRIVER_VERSION,
```

Comment:
what about using 
```
services: [
        ['selenium-standalone', { drivers: {  chrome: true } }]
    ]
```
from https://webdriver.io/docs/selenium-standalone-service.
In case you'll use this option, you can remove the script `GetChromeDriverVersion.js`